LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event.https.html

```diff
@@ -147,6 +147,28 @@
             'Service Worker should respond with an empty resulting client id for subresource requests');
         });
   }, 'Service Worker responds to fetch event with the correct resulting client id');
+  
+  promise_test(t => {
```

Comment:
Is this a test resync from upstream wpt ? If not, you will need to upstream those changes.

Reply:
This is a new test that I wrote. Ok, I will look into how to upstream it.